lib/include/openamp/virtio_drv.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * $FreeBSD$
+ */
+
+#ifndef _VIRTIO_DRV_H_
+#define _VIRTIO_DRV_H_
+
+#if defined(WITH_VIRTIO_MMIO_DRV)
+#include <openamp/virtio_mmio.h>
+
+#define virtio_drv_setup_virtqueue virtio_mmio_setup_virtqueue
```

Comment:
it's better to use ths patch:
https://github.com/OpenAMP/open-amp/pull/489/commits/8c60cb68fd76fa6d6083f0a27899efbacffaff53#diff-5fa1a1ad9533f2cdd8baf2b7732eed5bd82c85e19bb98b700d4417f1fd527d3aR202